server/app/controllers/me-controller.js

```diff
@@ -76,6 +76,8 @@ async function updateMe(req, res) {
         const { error } = schema.validate(newUser);
         if (error) throw createUserError('BadRequest', error.message);
 
+        if (newUser.barman.active) throw createUserError('BadRequest', 'Active field is not allowed throw PUT /me');
```

Comment:
We should check for undefined

Reply:
Not in the controller, cleanObject is only used in services because of sequelize

Reply:
newUser.barman can not be undefined (because of if statement on line 72). So, it is not a problem :)

Reply:
Actually I was talking about checking the field `active` because we don't want to have something set to `false`...

But this check can be done throught joi (prevent the field `active` in this function)

Reply:
I've tested:
- If `active` field is present (set to true or false): server throw `BadRequest, Active field is not allowed throw PUT /me` error because active field is compute as a string.
- If `active` field is undefined: server throw a `BadRequest, child \"active\" fails because [\"active\" must be a boolean]` error

So what do you advice ?